bin/compile

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env bash
 # bin/compile <build-dir>
+GHOSTSCRIPT_VERSION="$(cat $BUILD_DIR/.ghostscript_version)"
 
-echo "-----> Installing Ghostscript 9.22 using build directory $1"
+echo "-----> Installing $GHOSTSCRIPT_VERSION using build directory $1"
```

Comment:
Installing Ghostscript $GHOSTSCRIPT_VERSION

Reply:
I've put `GHOSTSCRIPT_VERSION="$(cat $BUILD_DIR/.ghostscript_version)"` before `echo`.

Reply:
Anyway I can change it.

Reply:
@renanigt you are right, it wouldn't :)